WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -139,3 +141,36 @@ extension RegisterDomainSuggestionsViewController: NUXButtonViewControllerDelega
         self.navigationController?.pushViewController(controller, animated: true)
     }
 }
+
+// MARK: - DomainCreditRedemptionSuccessViewControllerDelegate
+
+extension RegisterDomainSuggestionsViewController: DomainCreditRedemptionSuccessViewControllerDelegate {
+
+    /// Handles the dismission of the epilogue screen at the end of a successful domain registration, along with popping to the domains dashboard
+    func continueButtonPressed() {
+
+        dismiss(animated: true) { [weak self] in
+            if let popController = self?.navigationController?.viewControllers.first(where: {
+                                                                                $0 is UIHostingController<DomainsDashboardView>
+            }) ?? self?.navigationController?.topViewController {
+                self?.navigationController?.popToViewController(popController, animated: true)
+            }
+
+            guard let email = self?.accountEmail() else {
+                return
+            }
+            let title = String(format: NSLocalizedString("Verify your email address - instructions sent to %@",
+                                                         comment: "Notice displayed after domain credit redemption success."), email)
+            ActionDispatcher.dispatch(NoticeAction.post(Notice(title: title)))
```

Comment:
Just double checking with you: I imagine this isn't fully implemented yet, is that correct?

Reply:
We can put a TODO here: this notice might need to change. This is what we are currently doing for the existing domain registration

Reply:
Actually, as discussed, removing this notice